docs/en/src/derived_fields.md

```diff
@@ -40,7 +40,7 @@ struct Query;
 
 #[Object]
 impl Query {
-    #[graphql(derived(name = "date_rfc3339", into = "DateRFC3339"))]
+    #[graphql(derived(name = "duration_rfc3339", into = "DateRFC3339"))]
     async fn duration_rfc2822(&self, arg: String) -> DateRFC2822 {
```

Comment:
I think `Query::duration_rfc2822` should be renamed to `Query::date_rfc2822`

Reply:
Ok, will fix it.

Reply:
@sunli829  i have renamed `Query::duration_rfc2822` to `Query::date_rfc2822`

Reply:
The field `duration_rfc2822` should also be renamed to `date_rfc2822` 🙂

Reply:
I got it! Thank you @sunli829 for clarifying , I have fixed those changes.